health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -214,14 +468,14 @@ const onConfirmClick=()=>{
                                     <div style={{ fontWeight: "600", fontSize: "1.2rem" }}>
                                         {/* {item?.boundaryType} */}
-                                        {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase().replace(/\s+/g, "_"))}`}
+                                        {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase())}`}
                                     </div>
                                     <div style={{ height: "1rem" }}></div>
                                     <Card type={"primary"} variant={"form"} className={"question-card-container"} >
                                         <div style={{ display: "flex", gap: "2rem" }}>
                                             <Svgicon />
                                             <div style={{ display: "flex", alignItems: "center", fontWeight: "600" }}>
-                                                {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase().replace(/\s+/g, "_"))}`}{"-geojson.json"}
+                                                {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase())}`}{"-geojson.json"}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)